dataactvalidator/validation_handlers/validator.py

```diff
@@ -846,26 +847,41 @@ def validateFileBySql(submissionId,fileType,interfaces):
              values in fields involved
              row number
         """
-        # Skip Sql validation
-        return []
         # Pull all SQL rules for this file type
         fileId = interfaces.validationDb.getFileId(fileType)
         rules = interfaces.validationDb.session.query(RuleSql).filter(RuleSql.file_id == fileId).filter(RuleSql.rule_cross_file_flag == False).all()
         errors = []
         # For each rule, execute sql for rule
         for rule in rules:
-            failures = interfaces.validationDb.connection.execute(rule.rule_sql.format(submissionId))
+            failures = interfaces.stagingDb.connection.execute(rule.rule_sql.format(submissionId))
             # Build error list
             for failure in failures:
                 row = failure["row_number"]
-                errorMsg = rule.rule_error_msg
-                values = ""
-                fieldNames = ""
-                for field in failure:
+                errorMsg = rule.rule_error_message
+                values = None
+                fieldNames = None
+                # failure is a RowProxy, which does not quite act the same as a dict
+                for field in failure.keys():
```

Comment:
This is more about coding style and syntax preference, but it would be possible to avoid explicitly looping through the columns in the query result and having to do different formatting for the first value of the `values` and `fieldNames` variables. 

The cross-file processing does this by using a few list comprehension: https://github.com/fedspendingtransparency/data-act-broker-backend/blob/db-429-cross-file-sql/dataactvalidator/validation_handlers/validator.py#L47

(I'm a Python list comprehension fan)